src/api/comment/content-types/comment/lifecycles.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+
+    async afterCreate(event) {
+
+        const ideaCard = event.result.idea_card
+        const ideaName = event.result.idea_card.ideaName
+        const commentAuthor = event.result.author
+        const commentText = event.result.text
+        const timeCreated = event.result.createdAt
```

Comment:
maybe if we use object destructing here it'd be better

Reply:
Like `const {ideaCard, timeCreated} = event.result`.

Reply:
@Enjoy2Live You are right! Done it!